tests/python_tests/test_llm_pipeline_static.py

```diff
@@ -132,20 +132,18 @@ def test_max_number_of_tokens():
     assert len(encoded_results.tokens[0]) == num_tokens
 
 
-# FIXME: Known problem, output differs from stateful pipeline starting from 3rd prompt!
 @pytest.mark.skipif(sys.platform in ["darwin", "linux"], reason="Not supposed to work on mac. Segfault on linux CI")
-@pytest.mark.skip(reason="JIRA-144780: Output differs from stateful pipeline")
 @pytest.mark.precommit
 @pytest.mark.nightly
-def test_chat_generation(model_descr):
+def test_chat_generation():
     questions = [
         '1+1=',
         'What is the previous answer?',
         'Why is the Sun yellow?',
         'What was my first question?'
     ]
 
-    model_path = get_chat_models_lists()[0][1]
+    model_path = get_chat_models_list()[0][1]
```

Comment:
it's not a model path, it's a `model_id`. E.g. from CI error we can see:
```
model_path = WindowsPath('Qwen2-0.5B-Instruct'), device = 'CPU'
```
which means model **is not** even converted by Optimum

In other places, it's used like:
```python
    pipe = read_model(get_models_list()[0])[4]
```
where read model converts model and created pipe on to top it.

So, the question is - have you run tests locally? does they even magically pass?

Reply:
`get_chat_models_list()` returns `List[tuple(model_id, model_path)]`
https://github.com/openvinotoolkit/openvino.genai/blob/master/tests/python_tests/ov_genai_test_utils.py#L106

I'd assume `get_chat_models_list()[0][1]` is the `path` of the `first` model, but probably I'm wrong... But at least for `get_models_list()` it works fine: https://github.com/openvinotoolkit/openvino.genai/blob/master/tests/python_tests/test_llm_pipeline_static.py#L37

Perhaps that problem is that I need to call `read_model` explicitly at least once, let me try it.





>So, the question is - have you run tests locally? do they even magically pass?

I've used my own list of models available on local setup, so yes, I didn't check if this machinery works with default models